WORKING_GROUPS.md

```diff
@@ -0,0 +1,173 @@
+# IPFS Project Working Groups
+
+## Index
+
+- [Description](#description)
+- [Working Group Lifecycle](#working-group-lifecycle)
+- [Current Working Groups](#current-working-groups)
+- [Other](#other)
+
+## Description
+
+The IPFS Project Working Groups are teams of people that are appointed to research, develop and deploy new work under the scope that the working group is focused. This structure is designed to provide clarity and direction to the project, enabling individual contributors to focus their time and energy on the areas they are most interested.
+
+Each Working Group has its own cadence, lifecycle and objectives.
+
+## Working Group Lifecycle
+
+TODO: Define how a working group gets created, evaled and how to decide if its purpuse has been fulfilled and should be dismantled
+
+## Current Working Groups
+
+### Release
+
+The Release Working Group manages the releases for go-ipfs, js-ipfs and its internal dependencies.
+
+Responsibilities include:
+
+- Structure the released process.
+- Define the release Roadmap.
+- Decide what gets into each release.
+- Test releases.
+- Announce and gather feedback from the community on each release.
+
+### Specifications
+
+The Specifications Working Group coordinates and participates on the specification writting process.
+
+Responsibilities include:
+
+- Ensure that specifications are up to date.
+- Enable developers and researches to contribute to the specification writting process with minimum overhead
+- That implementations can be compliant to the specs through formal testing (PDD and proofs).
+- Facilitate discussion around each spec making sure that proposals are followed up.
+
+### Integration with Web Browsers
+
+The Integration with Web Browsers Working Group designs and implement Browser integrations, Web Extensions, Service Workers and any other strategy that contributes to IPFS being integrated with the Web today.
+
+Responsibilities include:
+
+- Browser integrations (Firefox, Chrome, Brave and others).
+- IPFS Web Extension.
+- IPFS Service Worker.
+- IPFS Web Worker.
+- Collaborate with with Browser vendors.
+- Identify blockers and design constraints of each integration, figuring out new ways to go around those limitations.
+- Increase the adoption of the Distributed Web by making it easy for users to access it.
+
```

Comment:
- Define specifications for address schemes and advocate for those schemes to be adopted

Reply:
🌟